addons/web/static/tests/views/search_view_tests.js

```diff
@@ -802,6 +802,78 @@ QUnit.module('Search View', {
         actionManager.destroy();
     });
 
+    QUnit.module('Search View Rendering');
+
+    QUnit.test('fields and filters with groups/invisible attribute are not always rendered but activable as search default', function (assert) {
+        assert.expect(15);
+
+        var actionManager = createActionManager({
+            actions: this.actions,
+            archs: this.archs,
+            data: this.data,
+            debug: true,
+        });
+
+        this.archs['partner,10,search'] =
+            "<search>" +
+                "<field name=\"display_name\" string=\"Foo B\" invisible=\"1\"/>" +
+                "<field name=\"foo\" string=\"Foo A\"/>" +
+                "<filter name=\"filterA\" string=\"FA\" domain=\"[]\"/>" +
+                "<filter name=\"filterB\" string=\"FB\" domain=\"[]\" invisible=\"1\" modifiers=\"{&quot;invisible&quot;: true}\"/>" +
```

Comment:
the `modifiers` attribute should be automatically generated by the MockServer: https://github.com/odoo/odoo/blob/12.0/addons/web/static/tests/helpers/mock_server.js#L283

If it doesn't work, we should understand why and fix it, probably (we want to write arch like we would really do, to be as close as possible as what happens "in real").

Reply:
done (and the "modifiers" are correctly added)